src/components/Card.tsx

```diff
@@ -1,28 +1,18 @@
 import React, { HTMLProps } from 'react';
 import classNames from 'classnames';
-import { GlobalDispatch, GlobalState, useGlobalDispatch, useGlobalSelector } from '../redux/global';
 
-export const Card: React.FC<HTMLProps<HTMLDivElement>> = ({ className, children, ...props }) => {
-    const { theme } = useGlobalSelector((state: GlobalState) => ({
-        theme: state.config.theme,
-    }));
+export const Card: React.FC<HTMLProps<HTMLDivElement>> = ({ className, children, ...props }) => (
 
-    const { colors: { text, padding } } = theme;
-
-    return (
-
-        <div
-            style={{ color: text, background: padding }}
-            className={
-                classNames(
-                    'rounded-xl drop-shadow px-5 py-4',
-                    { 'cursor-pointer hover:-translate-y-1 hover:drop-shadow-lg transition duration-200': props.onClick },
-                    className,
-                )
-            }
-            {...props}
-        >
-            {children}
-        </div>
-    );
-};
+    <div
+        className={
+            classNames(
+                'rounded-xl bg-theme-padding text-theme-text drop-shadow px-5 py-4',
+                { 'cursor-pointer hover:-translate-y-1 hover:drop-shadow-lg transition duration-200': props.onClick },
```

Comment:
You kinda overwrote it during conflicts resolution, it should be `hover:bg-silver-700` for hover animation.

Reply:
will add back the hover but hard coded colors have been removed otherwise it won't be adjustable

Reply:
Yeah, no problems at an all. I just wanted to get rid of animations that cause CLS.